cloud-data-protection/src/components/logout.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import './logout.css';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
import "./logout.css";
```